resources/views/layouts/password.blade.php

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
```

Comment:
Please consider to use `@extends('layouts.minimal')` instead of building all the template base